boards/arm/mimxrt1010_evk/doc/index.rst

```diff
@@ -191,6 +191,107 @@ see the following message in the terminal:
 
     Hello World! mimxrt1010_evk
 
+MCUXpresso Config Tool
+======================
+
+A ``mimxrt1010_evk.mex`` file is included. This file was used to generate the clock
+initialization code and can be used as a starting point to tweak the clock configuration.
+This could be useful for different boards that are based on i.MX RT1010.
+
+NOTE: The MCUXpresso Config Tool currently generates a ``.c`` file with the clock configuration.
+Considering options on leveraging this tool in the future to generate a devicetree compatible file.
+
+Below is the clock configuration at platform initialization.
+
+- On-chip 24MHz oscillator is enabled
```

Comment:
As requested in my review, I rather expect a "Clock start up configuration" chapter.

This is not that I don't want MCUXpresso to be mentioned or highlighted, but the main point here is to achieve a correct clock configuration, and that what we expect to be detailed following this change. MCUXpresso is only a way to achieve that (which will soon be replaced by device tree), and, from what you exposed during dev-review meeting, there are no plans to extend use of this tool to generate more code, so the code generation dedicated part would go away at one point, but a "Clock start up configuration" chapter would remain.

Reply:
@erwango, do you have any examples you can point us too? 

I'm asking because we have a lot of documentation about clocks in our reference manuals that I don't really want to reproduce here (fairly long) so want to see what others do in this area.

Reply:
I have moved the clock details to a new section.